.travis.yml

```diff
@@ -11,9 +11,22 @@ matrix:
   fast_finish: true
 
 env:
+  global:
+    - RUN_PHPSTAN="FALSE"
   matrix:
-    - LOWEST_DEPS=""
-    - LOWEST_DEPS="--prefer-lowest"
+    - PREFER_LOWEST=""
```

Comment:
In other repos this env var is called `PREFER_LOWEST` so I made it the same name here.